person.rb

```diff
@@ -2,7 +2,7 @@ class Person
   attr_accessor :name, :age
   attr_reader :id
 
-  def initialize(id, age, name = 'Unknown', parent_permission = true)
+  def initialize(id, age, name: 'Unknown', parent_permission: true)
     @id = id
```

Comment:
- [OPTIONAL] Following the best practices,  I'll suggest generating the id randomly, this is the best practice as ID should be unique and not null, you can have a code like this. `@id `= Random.rand(1..1000)``